governance/votes/Executive vote - June 13, 2024.md

```diff
@@ -0,0 +1,169 @@
+|-----------------|-----------------------------------------------------------------------------------------------------------------------|--------------------|--------------|
+| IamMeeoh        | [0x47f7A5d8D27f259582097E1eE59a07a816982AE9](https://etherscan.io/address/0x47f7A5d8D27f259582097E1eE59a07a816982AE9) | KISS               | 12.51        |       
+| DAI-Vinci       | [0x9ee47F0f82F1A6F45C4E1D25Ce95C321D8C8356a](https://etherscan.io/address/0x9ee47F0f82F1A6F45C4E1D25Ce95C321D8C8356a) | KISS               | 12.51        |       
+| opensky         | [0xf44f97f4113759E0a57756bE49C0655d490Cf19F](https://etherscan.io/address/0xf44f97f4113759E0a57756bE49C0655d490Cf19F) | KISS               | 12.51        |       
+| Res             | [0x8c5c8d76372954922400e4654AF7694e158AB784](https://etherscan.io/address/0x8c5c8d76372954922400e4654AF7694e158AB784) | Resiliency         | 12.51        |       
+| Harmony         | [0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3](https://etherscan.io/address/0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3) | Resiliency         | 12.51        |       
+| Libertas        | [0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f](https://etherscan.io/address/0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f) | Resiliency         | 12.51        |       
+| seedlatam.eth   | [0xd43b89621fFd48A8A51704f85fd0C87CbC0EB299](https://etherscan.io/address/0xd43b89621fFd48A8A51704f85fd0C87CbC0EB299) | Sovereign Finance  | 12.51        |       
+| 0xRoot          | [0xC74392777443a11Dc26Ce8A3D934370514F38A91](https://etherscan.io/address/0xC74392777443a11Dc26Ce8A3D934370514F38A91) | Sovereign Finance  | 12.51        |
+		
+### Launch Project Funding
+		
+- **Authorization:** [Accessibility Scope 9.1](https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/17)
+		
+If this executive proposal passes, the following transfers to the Launch Project will be made:
+		
+- Transfer **5,000,000 DAI** to the Launch Project at [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
+- Transfer **450 MKR** to the Launch Project at [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
+		
+### Spark Proxy Spell
+		
+- **Authorization:** [Governance Poll](https://vote.makerdao.com/polling/QmPmVeDx)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/may-31-2024-proposed-changes-to-sparklend-for-upcoming-spell/24413)
+		
+If this executive proposal passes, a Spark proxy spell at [$address]($etherscan_link) will be triggered. The expected contents are as follows:
+		
+#### Onboard Etherfi weETH to SparkLend on Mainnet
+		
+weETH will be onboarded to SparkLend on mainnet with the following parameters:
```

Comment:
Capital letter for "Mainnet"

Reply:
In the past this has only been capitalised in titles. Please consider for style guide inclusion, but it should not block approval of this copy.